tests/phpunit/tests/test-slugs.php

```diff
@@ -3,13 +3,15 @@
 class Slugs_Test extends PLL_UnitTestCase {
 
 	/**
-	 * @param WP_UnitTest_Factory $factory
+	 * @param PLL_UnitTest_Factory $factory
+	 * @return void
 	 */
-	public static function wpSetUpBeforeClass( WP_UnitTest_Factory $factory ) {
-		parent::wpSetUpBeforeClass( $factory );
+	public static function pllSetUpBeforeClass( PLL_UnitTest_Factory $factory ) {
+		parent::pllSetUpBeforeClass( $factory );
 
-		self::create_language( 'en_US' );
-		self::create_language( 'fr_FR' );
+		$factory->language->create( array( 'locale' => 'en_US' ) );
+		$factory->language->create( array( 'locale' => 'fr_FR' ) );
+		self::$model->options['default_lang'] = 'en'; // Otherwise static model isn't aware of the created languages...
```

Comment:
I don't like the idea that we would need to do this in all test classes. Maybe we already discussed the reason but I don't remember. Could you explain?

Reply:
Removed in d355d880.
However I kept it in `Static_Pages_Test` because the way some tests are initialized prevented me to used a concrete model. Maybe we could look at it later.